pants.toml

```diff
@@ -114,7 +114,7 @@ branch_notes = """
 pytest_plugins.add = [
   # TODO(#8651): We need this until we switch to implicit namespace packages so that pytest-cov
   #  understands our __init__ files. NB: this version matches 3rdparty/python/requirements.txt.
-  "setuptools==47.3.1",
+  "setuptools==49.1.0",
```

Comment:
You can remove this now, I think. We no longer use namespace packages with Pants.